sound/hda/intel-dsp-config.c

```diff
@@ -471,6 +471,26 @@ static const struct config_entry acpi_config_table[] = {
 		.acpi_hid = "808622A8",
 	},
 #endif
+/* Broadwell */
+#if IS_ENABLED(CONFIG_SND_SOC_INTEL_CATPT)
+	{
+		.flags = FLAG_SST,
+		.acpi_hid = "INT3438"
+	},
+#endif
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_BROADWELL)
+	{
+		.flags = FLAG_SOF,
+		.acpi_hid = "INT3438"
+	},
+#endif
```

Comment:
not actually proposing to do this, but just out of curiosity - the above could also have been written as
```
#if IS_ENABLED(CONFIG_SND_SOC_INTEL_CATPT) || IS_ENABLED(CONFIG_SND_SOC_SOF_BROADWELL)
	{
		.flags = 0
#if IS_ENABLED(CONFIG_SND_SOC_SOF_BROADWELL)
			| FLAG_SOF
#endif
#if IS_ENABLED(CONFIG_SND_SOC_INTEL_CATPT)
			| FLAG_SST
#endif
			,
		.acpi_hid = "INT3438"
	},
#endif
```
this would be too ugly even for me to propose, even though it would save one entry in the array :-)

Reply:
it's easier to manage if we deal with priority order at the higher level with clearer large blocks.